inc/commondbchild.class.php

```diff
@@ -861,6 +861,9 @@ function affectChild($id, $items_id = 0, $itemtype = '') {
    }
 
    public static final function getItemField($itemtype): string {
+      if (preg_match('/^Rule.+/', $itemtype)) {
```

Comment:
Maybe:
```suggestion
      if ($itemtype instanceof Rule) {
```

Reply:
instanceof will not work on a string, you will have to use `is_a()` function

Reply:
`is_a` did not work, I do not understand why... but Ive used `is_subclass_of` that did the trick :)